examples/swift-ios/bugsnag-example.xcodeproj/xcshareddata/xcschemes/bugsnag-example.xcscheme

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1150"
+   version = "1.3">
+   <BuildAction
```

Comment:
it looks like the schemes have been duplicated when opening the project as there are two objective-c-osx targets